src/App.tsx

```diff
@@ -261,7 +298,6 @@ const App = (): JSX.Element => {
         const newAddress = matchedAccount ? address : accounts[0].address;
 
```

Comment:
Just a note: Be careful when resolving merge conflict because I changed these lines in recently merged PR, so it's not caused just by linting :) https://github.com/interlay/interbtc-ui/blob/5eae28ce7475912ed60f5aea91ffd65678627a94/src/App.tsx#L194-L206

Reply:
Thanks. I took care of it.